lighthouse-core/test/audits/lcp-lazy-loaded-test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const LargestContentfulPaintLazyLoaded =
+  require('../../audits/lcp-lazy-loaded.js');
+
+/* eslint-env jest */
+describe('Performance: lcp-lazy-loaded audit', () => {
+  it('correctly surfaces the lazy loaded LCP element', async () => {
```

Comment:
could we get coverage of the viewport notapplicable case and the passing case as well?